Doc/library/textwrap.rst

```diff
@@ -17,31 +17,69 @@ If you're just wrapping or filling one or two text strings, the convenience
 functions should be good enough; otherwise, you should use an instance of
 :class:`TextWrapper` for efficiency.
 
-.. function:: wrap(text, width=70, **kwargs)
+.. function:: wrap(text, width=70, initial_indent="", subsequent_indent="", \
+                   expand_tabs=True, replace_whitespace=True, \
+                   fix_sentence_endings=False, \
+                   break_long_words=True, drop_whitespace=True, \
+                   break_on_hyphens=True, tabsize=8, \
+                   *, max_lines=None)
 
    Wraps the single paragraph in *text* (a string) so every line is at most
    *width* characters long.  Returns a list of output lines, without final
-   newlines.
+   newlines.  *width* defaults to ``70``.
 
-   Optional keyword arguments correspond to the instance attributes of
-   :class:`TextWrapper`, documented below.  *width* defaults to ``70``.
+   See :class:`TextWrapper` for documentation on all additional optional
+   arguments:
 
-   See the :meth:`TextWrapper.wrap` method for additional details on how
-   :func:`wrap` behaves.
+   * :attr:`.initial_indent`
+   * :attr:`.subsequent_indent`
+   * :attr:`.expand_tabs`
+   * :attr:`.replace_whitespace`
+   * :attr:`.fix_sentence_endings`
+   * :attr:`.break_long_words`
+   * :attr:`.drop_whitespace`
+   * :attr:`.break_on_hyphens`
+   * :attr:`.tabsize`
+   * :attr:`.max_lines`
 
+   Note that :func:`wrap` accepts exactly the same keyword arguments as
```

Comment:
These 2 notes seem unnecessary.

Reply:
Ok, I agree; it was inherited from the previous documentation, but I got rid of it. See 13e025a22a5c79aa67648d133c85e93207c725e1